src/app/lab/dropdowns/model-modal.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+import { ClimateModel } from '../../models/climate-model';
+import { Project } from '../../models/project';
+import { ClimateModelService } from '../../services/climate-model.service';
+
+/*  Model Modal Component
+    -- Requires project input
+    -- Emits selected model
+    Expected use:
+        <model-modal
+            [project]="your_project">
+*/
+
+@Component({
+  selector: 'model-modal',
+  templateUrl: './model-modal.component.html'
+})
+export class ModelModalComponent implements OnInit {
+
+    @Input() project: Project;
+
+    public climateModels: ClimateModel[] = [];
```

Comment:
I think this is another component that could use better scoping of its inputs/outputs. Rather than passing the entire project, this component only needs the list of selected models to set its initial state. Then, if its output was something like: `onModelsSelected(models: ClimateModel[])` which fired when the modal was closed, you could also easily clean up #85 by handling this event differently in the LabComponent and AddEditProjectComponent.